### PR TITLE
Update packages and examples to use OpenAI v4 SDK

### PR DESCRIPTION
**Pull request by MaxLeiter**
In preparation for openai@4: https://github.com/openai/openai-node/releases

### CODE REVIEW

File: docs/pages/docs/getting-started.mdx [hunk 1]
```diff
@@ -63,10 +63,10 @@ We've written some code to get you started — follow the instructions below to
 
     #### Install Dependencies
 
-    Next, we'll install `ai` and `openai-edge`. The latter is preferred over the official OpenAI SDK due to its compatibility with Vercel Edge Functions.
+    Next, we'll install `ai` and `openai`, OpenAI's official JavaScript SDK compatible with the Vercel Edge Runtime.
```
Comment (rattrayalex) [inline docs/pages/docs/getting-started.mdx:66]: ❤️

File: docs/pages/docs/guides/providers/openai.mdx [hunk 2]
```diff
@@ -8,18 +8,25 @@ import { Steps, Callout } from 'nextra-theme-docs'
 
 Vercel AI SDK provides a set of utilities to make it easy to use OpenAI's API. In this guide, we'll walk through how to use the utilities to create a chat bot and a text completion app.
 
+<Callout>
+  Before the release of `openai@4`, we recommended using the `openai-edge`
+  library because of it's compatibility with Vercel Edge Runtime. The OpenAI SDk
```
Comment (rattrayalex) [inline docs/pages/docs/guides/providers/openai.mdx:13]:
```suggestion
  library because of it's compatibility with Vercel Edge Runtime. The OpenAI SDK
```

**Review by rattrayalex — COMMENTED** [review]
FWIW what I see lgtm! Thanks for doing this!

**Review by jaredpalmer — APPROVED** [review]

**Comment by jaredpalmer** [comment]
@MaxLeiter fast follow to fix the sveltekit errors in actions

**Pull request merged**